### PR TITLE
feat: Optimize the process of importing wallets via phrase seeds

### PR DESCRIPTION
**Pull request by devchenyan**
issue: https://github.com/Magickbase/neuron-public-issues/issues/381

https://github.com/nervosnetwork/neuron/assets/11756869/52247fce-3ed7-4728-ac69-c51f77fe2288

image

### CODE REVIEW

File: packages/neuron-ui/src/components/WalletWizard/hooks.ts [hunk 1]
```diff
@@ -7,6 +7,17 @@ export const useInputWords = () => {
...
+      if (Number(idx) === 0) {
+        const list = value
+          .trim()
+          .replace(/[^0-9a-z]+/g, ' ')
+          .split(' ')
+        if (list.length === 12) {
+          setInputsWords(list)
+          return
+        }
+      }
```
Comment (yanguoyu) [inline packages/neuron-ui/src/components/WalletWizard/hooks.ts:19]: I do not comprehend the function of this segment of code.

Comment (devchenyan) [reply packages/neuron-ui/src/components/WalletWizard/hooks.ts:19]: > I do not comprehend the function of this segment of code.

image

Comment (yanguoyu) [reply packages/neuron-ui/src/components/WalletWizard/hooks.ts:19]: @Danie0918 Maybe we should add more tips to tell the user that the first input can enter all of the seed phrases

Comment (Danie0918) [reply packages/neuron-ui/src/components/WalletWizard/hooks.ts:19]: > @Danie0918 Maybe we should add more tips to tell the user that the first input can enter all of the seed phrases

UI：https://www.figma.com/design/6XNoimRDbFTTNm016rbIdU/Magickbase?node-id=319-4539&t=a0LYiUc0DrgWeXgO-0

Added. @devchenyan

File: packages/neuron-ui/src/widgets/MnemonicInput/index.tsx [hunk 2]
```diff
@@ -1,20 +1,90 @@
...
+    [setFocusIndex]
+  )
+
+  const options = useMemo(
+    () => mnemonicWordList.filter(item => item.indexOf(inputsWords[focusIndex]) === 0),
+    [focusIndex, inputsWords]
+  )
+
+  const onDocumentClick = useCallback(
+    (e: any) => {
```
Comment (yanguoyu) [inline packages/neuron-ui/src/widgets/MnemonicInput/index.tsx:60]: It's better to avoid using `any`

**Review by yanguoyu — APPROVED** [review]

**Comment by Danie0918** [comment]
@homura  Please have a review.

**Review by homura — APPROVED** [review]

**Comment by silySuper** [comment]
截屏2024-07-12 16 36 34

When copy in first input ,It is better to not confirm first word again.

**Comment by Danie0918** [comment]
> 截屏2024-07-12 16 36 34
> When copy in first input ,It is better to not confirm first word again.

This can be ignored.

**Review by silySuper — APPROVED** [review]

**Pull request merged**